### ISSUE TITLE
Clean up modelToXML?

### ISSUE THREAD

**Issue by sorawee** [cemcutting/Forge#29]
Just curious: why don't we use the `xml` collection to convert Racket value to XML string? It's easier to manipulate `xexpr`, less error-prone, and probably more efficient than using `string-append` (which could theoretically take quadratic time in the size of the datum).

**Comment by MLavrentyev** [comment]
Yes - this is good to put on our todo list for when we go through and do a clean-up/refactoring. It was like this originally but got lost in the switch to/from json.

**Comment by asvarga** [comment]
I agree, I used it for the other direction (parsing xml models to partial instances) and it was way less painful. The bug patched today was caused by string difficulties.
